common/shlibs

```diff
@@ -2519,6 +2519,7 @@ libsignon-qt5.so.1 signond-8.58_1
 libsignon-plugins-common.so.1 signond-8.58_1
 libsignon-extension.so.1 signond-8.58_1
 libj4status-plugin.so.0 j4status-0.1_1
+libbemenu.so.1 bemenu-1.0.0_1
```

Comment:
do you plan to add something that will use it in the near future ? if not please don't add common/shlibs entries

Reply:
Ok, I will remove it.